src/tsconfig.spec.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "../tsconfig.json",
     "compilerOptions": {
+        "module": "commonjs",
```

Comment:
this fix this:
https://github.com/angular/angular-cli/issues/17207

Reply:
OHHHHH!! I was wondering how you did to spy `window.fetch` because I had to skip a bunch of tests because of it. Thanks, great find.